src/main/java/org/aesh/parser/LineParser.java

```diff
@@ -143,7 +143,6 @@ else if (parseCurlyAndSquareBrackets && (c == CURLY_END || c == PARENTHESIS_END)
                 handleCurlyEnd(c);
             }
             else if (haveEscape) {
-                builder.append(BACK_SLASH);
```

Comment:
I think that this change impacts goes further than the fix. Any '\' character contained in non quoted text is removed by the parser. I don't think tha tthis is what we want. My understanding is as follow: 
1) Use '\' to escape a space character. In this case the '\' character is removed but no new word is created.
2) Any other usages, the \ character is kept.

The reported problem shows that we need to offer a way to escape operators too and have the '\' be removed from the parsed content in this case.

So I think that the fix should be narrower, something like:
 else if (haveEscape) {
                    //Escaping an operator?
                    if (!isQuoted()
                            && (currentOperator = matchesOperators(operators, text, index)) != OperatorType.NONE) {
                        // Do not add the \, that was a way to escape an operator.
                    } else {
                        builder.append(BACK_SLASH);
                    }
                    builder.append(c);
                    haveEscape = false;
                }

Reply:
ah, i missed this. good catch @jfdenise, yes we need this change.

Reply:
In this method there are no operators, so I just did undu the deletion of the backslash insertion.